server_structs/director_test.go

```diff
@@ -151,6 +151,7 @@ func TestConversion(t *testing.T) {
 
 	OAdConv := ConvertOriginAdV1ToV2(oAdV1)
 	oAdV2.GenerationID = OAdConv.GenerationID
+	oAdV2.Expiration = OAdConv.Expiration
```

Comment:
In other places you use your getters on these ads, but here the fields are accessed directly. If there's no need to use these interface methods, I'd rather see the simpler direct access approach.